Packs/CrowdStrikeFalcon/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,15 @@
+
+### Integrations
+- __CrowdstrikeFalcon__
+  - Added the following real-time response API commands:
+    - ***cs-falcon-run-get-command***
+    - ***cs-falcon-status-get-command***
+    - ***cs-falcon-status-command***
+    - ***cs-falcon-get-extracted-file***
+    - ***cs-falcon-list-host-files***
+    - ***cs-falcon-refresh-session***
+    
+  - Added the *target* argument to the ***cs-falcon-run-command*** command to support single and batch operations.
+  - Fixed entry context keys
+  - Fixed the ***cs-falcon-get-script*** command. A script entry returned from the command replaces the entry identifying with `ID` in `CrowdStrike.Script`.
```

Comment:
@kirbles19 i see it was dropped from the release notes, the last 2 bullets are breaking bc, so need to mention it